### PR TITLE
3 set up mkdocs documentation like in instructor aiinstructor js

### LINKED ISSUE #3: Set up MKDocs Documentation like in `instructor-ai/instructor-js`

### PR DESCRIPTION
**Pull request by sergiobayona**
The following docs were updated to reflect Instructor-rb:

- docs
    - blog
        - .authors.yml
        - index.md
    - concepts
        - patching.md
        - philosophy.md
        - schema.md
    - examples
        - action_items.md
        - action_itesm.png
        - index.md
    - overrides
        - main.html
    - contributing.md
    - help.md
    - index.md
    - installation.md

### CODE REVIEW

File: docs/examples/classification.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Text Classification
```
Comment (jxnl) [inline docs/examples/classification.md]: TODO: to ruby

File: docs/examples/content_moderation.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Incorporating OpenAI moderation validation
+
+The following code defines a schema to validate content using OpenAI's Moderation endpoint. Zod's `.superRefine()` is used to apply OpenAI's moderation after the compute. This moderation checks if the content complies with OpenAI's usage policies and flags any harmful content. Here's how it works:
+
+1. Initialize the OpenAI client and extend it with `Instructor`. This is not strictly necessary for this example, always recommended in order to leverage the full `Instructor` functionality.
+
+2. Define a Zod schema for our content, then super refine our `message` field with `moderationValidator(client)`. This means that after `message` is computed, it will be passed to `moderationValidator()` for validation. 
+
+```ts
```
Comment (jxnl) [inline docs/examples/content_moderation.md]: TODO: to ruby

File: docs/examples/query_decomposition.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    * Workflow automation
+    * Process optimization
+
+With the OpenAI Function Call model, you can customize the planning process and integrate it into your specific application to meet your unique requirements.
+
+## Defining the structures
+
+Let's define the necessary models to represent the query plan and the queries.
+
+```ts
```
Comment (jxnl) [inline docs/examples/query_decomposition.md]: TODO: to ruby

File: docs/examples/validated_citations.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
+# Example: Answering Questions with Validated Citations
```
Comment (jxnl) [inline docs/examples/validated_citations.md]: TODO: to ruby

**Review by jxnl — COMMENTED** [review]
lets delete the examples and ask the community to move stuff for now, and focus on install instructions and making sure stuff is working on install

**Comment by sergiobayona** [comment]
@jxnl okay I left the following placeholder files in the `docs/examples` folder:
- classification.md
- content_moderation.md
- query_decomposition.md
- self-correction.md
- validated_citations.md

**Pull request merged**